### PR TITLE
Feat: Add read asset guide

### PR DESCRIPTION
**Pull request by fhildeb**

### CODE REVIEW

File: docs/learn/dapp-developer/read-asset-data.md [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    }
+  }
+}
+```
+
+</details>
+
+:::note get and fetch
+
+The [`getData(...)`](../../tools/erc725js/classes/ERC725#getdata) function only retrieves the data keys values from the smart contract. In comparison, [`fetchData(...)`](../../tools/erc725js/classes/ERC725#fetchdata) will download and decode the content of a `VerifiableURI`, `JSONURL` and `AssetURL`.
```
Comment (Hugoo) [inline docs/learn/dapp-developer/read-asset-data.md:255]: `JSONURL` and `AssetURL` are deprecated - this guide should not mention them.

**Comment by Hugoo** [comment]
This should be emrged with this:

https://docs.lukso.tech/learn/expert-guides/read-asset-data

When we merge this PR, we should not have 2 "Read asset data" guides. But only one that takes the best out of the 2.

**Comment by Hugoo** [comment]
@fhildeb sorry I did a push force for my last commit as I rebased, you might need to git reset origin.

I sent an improvement with TODO left and an improved structure.

File: docs/learn/dapp-developer/read-asset-data.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Instead of providing a specific data key like `SupportedStandards:LSP4DigitalAss
 
 ## Detect the Token Type
 
-If both, contract and metadata standard are aligned, we can continue to determine the [LSP4 Token Type](https://docs.lukso.tech/standards/tokens/LSP4-Digital-Asset-Metadata/#types-of-digital-assets) in order to interpret the metadata correctly.
+If both, contract and metadata standard are aligned, we can continue to determine the [LSP4 Token Type](../../standards/tokens/LSP4-Digital-Asset-Metadata/#types-of-digital-assets) in order to interpret the metadata correctly.
```
Comment (fhildeb) [inline docs/learn/dapp-developer/read-asset-data.md:146]: `.md` missing on the link

Comment (Hugoo) [reply docs/learn/dapp-developer/read-asset-data.md]: nice catch ty

**Review by Hugoo — APPROVED** [review]

**Pull request merged**